internal/v1/api.yaml

```diff
@@ -1783,6 +1781,8 @@ components:
           $ref: '#/components/schemas/Installer'
         cacerts:
           $ref: '#/components/schemas/CACertsCustomization'
+        aap_registration:
+          $ref: '#/components/schemas/AAPRegistration'
```

Comment:
This is an API breaking change, will UI be ready when we merge this? It will go straight into stage.

Reply:
Is it?  It's just additive, isn't it?